### PR TITLE
added Get Plans API and two Payment WebHook events

### PR DESCRIPTION
**Pull request by svedule**
added Get Plans API and two Payment WebHook events

### CODE REVIEW

File: _posts/2017-05-11-integrating-with-billing.md [hunk 1]
```diff
@@ -267,6 +271,33 @@ Sample Response:
...
+#### (4) Get Plan Information
+
+All Plan details for a particular app(using packageName) can be obtained as follows:
+
+For eg:
+~~~
+Request URL - {endpoint}/apps/{packageName}/plans
+
+curl -X GET \
+  https://billing.poynt.net/apps/com.satyavedule.paymentfragment/plans \
```
Comment (eddhkim) [inline _posts/2017-05-11-integrating-with-billing.md]: do you want to use the appId to be consistent with pervious example?
`urn:aid:b326335b-ce7c-4482-80d4-109e0fe6f9d9`

File: _posts/2017-05-11-integrating-with-billing.md [hunk 2]
```diff
@@ -198,6 +198,8 @@ Sample Response:
 The Event types supported for receiving Billing Webhooks are:<br>
 <br>&nbsp;&nbsp;1)  <strong>_Subscription activation_{:.italic}</strong> ( `APPLICATION_SUBSCRIPTION_START` )
 <br>&nbsp;&nbsp;2)  <strong>_Subscription cancellation/end_{:.italic}</strong> ( `APPLICATION_SUBSCRIPTION_END` )
+<br>&nbsp;&nbsp;1)  <strong>_Subscription Payment Success_{:.italic}</strong> ( `APPLICATION_SUBSCRIPTION_PAYMENT_SUCCESS` )
```
Comment (eddhkim) [inline _posts/2017-05-11-integrating-with-billing.md]: shouldn't this be 3 and 4?

**Comment by eddhkim** [comment]
@svedule comments added

**Comment by eddhkim** [comment]
📝 👍

**Comment by leechenghan** [comment]
very nice

**Comment by leechenghan** [comment]
ship it

**Comment by eddhkim** [comment]
🚢 🇮🇹

**Pull request merged**